atlite/resource.py

```diff
@@ -53,10 +54,12 @@ def get_windturbineconfig(turbine):
         return get_oedb_windturbineconfig(turbine[len("oedb:") :])
 
     if isinstance(turbine, str):
-        if not turbine.endswith(".yaml"):
-            turbine += ".yaml"
 
-        turbine = WINDTURBINE_DIRECTORY / turbine
+        if not Path(turbine).exists():
+            if not turbine.endswith(".yaml"):
+                turbine += ".yaml"
+
+            turbine = WINDTURBINE_DIRECTORY / turbine
```

Comment:
Rather than presuming that if the Path does not exist that then it might be a pre-shipped one, we can check if it is a shipped config by checking:

```
turbine in windturbines
```

and then instead of manually constructing the path

```
turbine = windturbines[turbine]
```

as we have the turbines which are pre-shipped as well as their paths already read in `windturbineconfig`.